package/MDAnalysis/lib/util.py

```diff
@@ -295,7 +295,7 @@ def anyopen(datasource, mode='r', reset=True):
                     stream.reset()
                 except (AttributeError, IOError):
                     try:
-                        stream.seek(0L)
```

Comment:
Maybe it does. But I do not see why it would. Nevertheless, it does not break the test and I do not see in `io.IOBase` doc that `seek` requires a long integer.